modules/ROOT/pages/cassandra/cassandra-studio.adoc

```diff
@@ -0,0 +1,112 @@
+= Studio - Mule 4
```

Comment:
What is this topic for? task, reference, concept? And why does it mention Mule 4 but not Cassandra?

Reply:
It's supposed to be a Studio How to. Again, hoping  this goes up to Meera.